cmd/update-main_test.go

```diff
@@ -68,9 +68,9 @@ func TestGetCurrentReleaseTime(t *testing.T) {
 		{minioVersion2, minioBinaryPath2, releaseTime2, nil},
 		{"junk", minioBinaryPath2, releaseTime2, nil},
 		{"3.2.0", minioBinaryPath2, releaseTime2, nil},
-		{minioVersion2, "", time.Time{}, fmt.Errorf(errorMessage1)},
-		{"junk", "non-existing-file", time.Time{}, fmt.Errorf(errorMessage2)},
-		{"3.2.0", "non-existing-file", time.Time{}, fmt.Errorf(errorMessage2)},
+		{minioVersion2, "", time.Time{}, errors.New(errorMessage1)},
+		{"junk", "non-existing-file", time.Time{}, errors.New(errorMessage2)},
+		{"3.2.0", "non-existing-file", time.Time{}, errors.New(errorMessage2)},
```

Comment:
That's the only success cases.  How about `C:\non-existing-file` and `/usr/bin/non-existing-file` and existing file in the path with relative path like `ls` command from $PATH.

These tests are required to increase code coverage.